frontend/javascripts/oxalis/view/right-menu/meshes_view.js

```diff
@@ -131,6 +139,67 @@ class MeshesView extends React.Component<Props, { currentlyEditedMesh: ?MeshMeta
   };
 
   render() {
+    const moveToIsosurface = (seedPosition: Vector3) => {
+      Store.dispatch(setPositionAction(seedPosition));
+    };
+    const downloadButton = (segmentId: number) => (
+      <Tooltip title="download isosurface">
```

Comment:
We usually use title case for UI labels (https://titlecase.com/). Same for the other button's tooltips :)

Reply:
Good to know, thanks!